frontend/packages/data-portal/app/components/Breadcrumbs.tsx

```diff
@@ -142,23 +155,25 @@ export function Breadcrumbs({
         {variant === 'deposition' ? (
           <Breadcrumb text={t('deposition')} />
         ) : (
-          <Breadcrumb
-            text={
-              variant === 'dataset'
-                ? `${t('dataset')}`
-                : `${t('dataset')}: ${data.title}`
-            }
-            link={singleDatasetLink}
-            className="overflow-ellipsis overflow-hidden flex-initial"
-            type={BreadcrumbType.SingleDataset}
-            datasetId={data.id}
-          />
+          <Tooltip tooltip={variant === 'dataset'? `${t('dataset')}` : `${t('dataset')}: ${data.title}`} className='overflow-hidden overflow-ellipsis'>
+            <Breadcrumb
+              text={
+                variant === 'dataset'
+                  ? `${t('dataset')}`
+                  : `${t('dataset')}: ${data.title}`
+              }
+              link={singleDatasetLink}
+              className="overflow-ellipsis overflow-hidden flex-initial"
+              type={BreadcrumbType.SingleDataset}
+              datasetId={data.id}
+            />
+          </Tooltip>
         )}
 
-        {variant === 'run' && (
+        {activeBreadcrumbText && (
```

Comment:
If you follow this logic, it should instead make the default be t('run') if there activeBreadCrumbText is undefined and variant is run

Reply:
I reverted back the change the `variant === 'run'` 
If we don't have `activeBreadcrumbText` it doesn't show anything, let me know what can I show in this case? 
Should i show t('run')? 

@seankmartin